bin/generate-tests.php

```diff
@@ -263,7 +264,10 @@ public function test(): void
     {{{ skip-on-binary }}
         $process = $this->runTask([{{ args }}]{{ cwd }}{{ needRemote }}{{ needResetVendor }});
 
-        $this->assertSame({{ exitCode }}, $process->getExitCode());
+        if ({{ exitCode }} !== $process->getExitCode()) {
+            throw new ProcessFailedException($process);
+        }
```

Comment:
It allows a better output when an test failed instead of phpunit only saying that number is different, at least we know why